src/cli.rs

```diff
@@ -773,18 +773,19 @@ fn init_commands<'a, C: CLI<'a>>() -> Vec<Command<'a, C>> {
                                 if wait == Some(true) {
                                     if let Err(err) = keystore.await_key_scan(&key.address()).await {
                                         cli_writeln!(io, "Error waiting for key scan: {}", err);
+                                        return
                                     }
-                                } else {
+                                } else{
                                     cli_writeln!(io,
                                         "Note: assets belonging to this key will become available
                                         after a scan of the ledger. This may take a long time. If
-                                        you have the owner memo for a record you want to use
+                                        you have the owner memo for a record you want to uses
                                         immediately, use import_memo.");
-                                    // Output both the public key and the address when loading a
-                                    // sending key.
-                                    cli_writeln!(io, "{}", key.pub_key());
-                                    cli_writeln!(io, "{}", UserAddress(key.address()));
                                 }
+                                // Output both the public key and the address when loading a
+                                // sending key.
+                                cli_writeln!(io, "{}", key.pub_key());
+                                cli_writeln!(io, "{}", UserAddress(key.address()));
```

Comment:
It's making sure to write the keys to console regardless of if you wait for the key scan or not.  Previously this would not print anything to console if you input "wait=true".   Without this change the tests don't work Also, if a real user loaded from a key file it would be difficult for them to then use the address and pub key because they aren't printed to the console.

Reply:
Oh, I see. That makes sense